packages/ui/src/components/va-image/_variables.scss

```diff
@@ -1,4 +1,6 @@
+:root,
 :host {
-  --va-image-position: absolute;
-  --va-image-object-position: 50% 50%;
+  --va-image-content-position: absolute;
```

Comment:
Any reason for this to be a CSS variable? I can't see any cases when user is going to change it to `relative`. I would remove all css variables from here.

Reply:
<img width="838" alt="image" src="https://user-images.githubusercontent.com/64714442/221231442-ce45a7dc-82e2-4811-8a30-8b18b65c31a0.png">

`content-position` is also a real case.

But i can agree that `content-width` is redundant.

Reply:
So there are cases when user want content position to be relative? Can you add a comment on top of css variable? We will show this later in docs.